commands/fetch.ts

```diff
@@ -38,22 +44,27 @@ exports.handler = async options => {
 
   trackCommandUsage('fetch', { mode: cmsPublishMode }, derivedAccountId);
 
+  const { assetVersion, staging } = options;
   try {
     // Fetch and write file/folder.
     await downloadFileOrFolder(
-      derivedAccountId,
+      derivedAccountId!,
```

Comment:
I'm not crazy about using the non null assertion operator, but I was trying to prevent modifying behavior because the yargs types are incorrect.  My understanding is `derivedAccountId` will always be defined

Reply:
This should be fixed when you leverage the CommonArgs type